src/routes/formsgSiteLaunch.ts

```diff
@@ -221,6 +224,13 @@ export class FormsgSiteLaunchRouter {
     await mailer.sendMail(requesterEmail, subject, html)
   }
 
+  sendMonitorCreationFailure = async (repoName: string): Promise<void> => {
```

Comment:
nit: perhaps can consider including the primary domain source too? Sometimes it might not be intuitive from the repo name.

Reply:
I realised this is already the primary domain source and not the repo name - renamed in https://github.com/isomerpages/isomercms-backend/pull/960/commits/8940b7333d8519cbeccda876ffc44dc384e1bc1f!